test/e2e/azure_test.go

```diff
@@ -839,15 +839,6 @@ var _ = Describe("Workload cluster creation", func() {
 					}
 				})
 			})
-
-			By("modifying OIDC issuer configuration", func() {
```

Comment:
If we don't keep the "actual" state in the status, then we have to wait to add this test back until we can use SDK v2 clients in the tests.